homeassistant/components/netatmo/climate.py

```diff
@@ -237,6 +238,10 @@ async def async_added_to_hass(self) -> None:
                 )
             )
 
+        registry = await self.hass.helpers.device_registry.async_get_registry()
+        device = registry.async_get_device({(DOMAIN, self._id)}, set())
+        self.hass.data[DOMAIN][DATA_DEVICE_IDS][self._home_id] = device.id
```

Comment:
Is it possible to have conflicting ids if they have multiple `netatmo` config entries?

Reply:
Nevermind, only one is allowed